lib/bitshares.js

```diff
@@ -198,6 +198,43 @@ export default class BitShares {
     }
   }
 
+    async memoDecode(memos) {
+        if (!this.memoKey)
+            throw new Error("Not set memoKey!");
+
+        let memo_text;
+        let pub_key = await PublicKey.fromPublicKeyString(memos.from);
+        let prv_key = this.memoKey;
+        // console.log("\nPrivate key:", prv_key.toWif());
+        console.log("Public key :", prv_key.toPublicKey().toString(), "\n");
+
+        if (prv_key) {
+            try {
+                memo_text = prv_key
+                    ? await Aes.decrypt_with_checksum(
```

Comment:
`if (prv_key)` and after that `prv_key ? ... : ...`? 
One condition excludes the other